lib/src/config/config_file/mod.rs

```diff
@@ -530,6 +532,12 @@ impl From<MempoolFilterSettings> for MempoolRules {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Default)]
+#[serde(deny_unknown_fields)]
+pub struct ValidatorWalletSettings {
+    pub key_file: Option<String>,
```

Comment:
You can move this to the `ValidatorSettings` and call it `wallet_key_file`.
That way we don't need a separate section in the config.